app-frontend/src/app/services/map/map.service.js

```diff
@@ -529,26 +530,92 @@ class MapWrapper {
         this.heldState = state;
         return this;
     }
+
+    setPlanetThumbnail(scene, persist, planetKey) {
```

Comment:
I'd rather modify the current `setThumbnail` method by adding flexibility than add a second way to add thumbnails. Adding a second special function like this unneccesarily expands the api surface.
I'd also rather not make the map depend on the planet service - better to make planetService have a setPlanetThumbnail function that uses the existing setThumbnail function.